django_celery_results/models.py

```diff
@@ -32,6 +32,18 @@ class TaskResult(models.Model):
     content_encoding = models.CharField(
         _('content encoding'), max_length=64,
     )
+    task_name = models.CharField(
+        _('task name'),
+        null=True, max_length=255
+    )
+    task_args = models.TextField(
+        _('task arguments'),
+        null=True
+    )
+    task_kwargs = models.TextField(
+        _('task keyword arguments'),
+        null=True
+    )
```

Comment:
Following the [Django recommendation](https://docs.djangoproject.com/en/1.10/ref/models/fields/#null) for string based fields, this should not be using `null=True` but instead `default='', blank=True`. Then only set the values when storing the result if given, the default will take care of the rest.